test/extensions/filters/http/wasm/wasm_filter_stress_test.cc

```diff
@@ -0,0 +1,234 @@
+#include "envoy/server/wasm_config.h"
+#include "test/stress/stress_test.h"
+
+using namespace Envoy::Stress;
+
+namespace Envoy {
+namespace Extensions {
+namespace Wasm {
+
+static const std::string STAT_PREFIX{"wasm_filter_stress_test"};
+
+// TODO add HTTP1 vs. HTTP2 X inbound vs outbound X IPv4 vs. IPV6 to parameterized test
+class WasmStressTest : public Stress::StressTest,
+                       public testing::TestWithParam<std::tuple<std::string, std::string>> {
+public:
+  // TODO parameterize
+  WasmStressTest()
+      : StressTest(Network::Address::IpVersion::v4, Http::CodecClient::Type::HTTP1,
+                   Http::CodecClient::Type::HTTP1) {}
+
+  virtual ~WasmStressTest() {}
+};
+
+INSTANTIATE_TEST_SUITE_P(RuntimesAndLanguages, WasmStressTest,
+                         testing::Combine(testing::Values(
+#if defined(ENVOY_WASM_V8) && defined(ENVOY_WASM_WAVM)
+                                              "envoy.wasm.vm.v8", "envoy.wasm.vm.wavm"
+#elif defined(ENVOY_WASM_V8)
+                                              "envoy.wasm.vm.v8"
+#elif defined(ENVOY_WASM_WAVM)
+                                              "envoy.wasm.vm.wavm"
+#endif
+                                              ),
+                                          testing::Values("cpp")));
+
+TEST_P(WasmStressTest, GrpcCalloutFailOpenHappyPath) {
+  Envoy::Logger::Registry::setLogLevel(spdlog::level::info);
```

Comment:
Does anyone know how to set this on the bazel commandline?

Reply:
Try `bazel test --define wasm=enabled //test/... -- -l info`

Reply:
Thanks @PiotrSikora